src/App.tsx

```diff
@@ -15,48 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum SortOptions {
+  Alphabetically = 'alphabetically',
+  Length = 'length',
+  Reset = 'reset',
+}
```

Comment:
```suggestion
export enum SortOptions {
  Alphabetically = 'alphabetically',
  Length = 'length',
  None = '',
}
```
reset from another state. it is not sort oprion